modules/fts/pages/fts-monitor.adoc

```diff
@@ -1,11 +1,104 @@
+
+Hold the mouse pointer on each chart to view the statistical details corresponding to the chart. Additionally, click the chart to expand the chart in full view.
+
+Couchbase provides statistics for the FTS indexes on the Dashboard page and the Search page.
+
+* Dashboard page
+* Search page
+
+=== Dashboard page
+
+To view the statistics on the Dashboard page:
+
+1. In the left pane, click *Dashboard*.
+2. On the Dashboard page, click the *Search indexes* drop-down list and select the FTS index you want to view the statistics.
+
+A graphical display of the statistics is as shown below:
+
+[#fts_fts_monitoring_statistics1]
+image::fts-monitoring-statistics1.png[,820,align=left]
+
+The statistics display the following graphical charts for FTS index:
+* Search Mutations Remaining: The no. of mutations not yet indexed.
+
+=== Search page
+
+To view the statistics on the Search page:
+
+1. In the left pane, click *Search*.
+2. On the Full Text Indexes page, click the full text search *index name* in the Index Name column.
+3. Click the *Search Index Stats* drop-down arrow to display the statistics.
+4. (Optional) Click the *interval* drop-down list and select the time interval to view the statistics. By default, you can view the statistics for every minute.
+
+A graphical display of the statistics is shown below:
+
+[#fts_fts_monitoring_statistics2]
+image::fts-monitoring-statistics2.png[,820,align=left] 
+
+The statistics display the following graphical charts:
+
+* Search Query Latency: The average time to run the search query and get a response.
+* Search Docs: The number of documents processed during the indexing.
+* Search Disk Size: The total size of disk space occupied by the fts index.
+* Search Disk Files: The number of index files on disk.
+* Search Memory Segments: The number of memory segments in the full text search index across all partitions.
+* Search Disk Segments: The number of disk segments in the full text search index across all resident disk partions.
+* Search Mutations Remaining: The number of mutations not yet indexed. 
+* Search Partitions: The number of actual index partitions.
+* Search Partitions Expected: The number of partitions expected to be scanned for the search query.
+* Search Records to Persist: The number of index records not yet persisted to the disk.
+* Search Index Rate: The rate (In Bytes per second) of indexing plain text.
+* Search Result Rate: The rate (In Bytes per second) of returning the search result.
+* Search Compaction Rate: The rate (In Bytes per second) of compaction of the index segments.
+* Search Query Rate: The number of queries run per second.
+* Search Query Error Rate: The number of queries per second (Including timeouts) that resulted in an error.
+* Search Slow Queries: The number of queries that run slowly (greater than 5 seconds to run) ) per second.
```

Comment:
G in capital case for (greater...) ?
In all other cases, starting letter is in caps)

Reply:
Updated